### DISCUSSION TITLE
error in data processing (nothing is displayed)

### DISCUSSION THREAD

**Discussion by vejobo9414larland** [spacecowboy/Feeder#241] [General]
Hello.
The application cannot process received responses for the following requests:
https://rutube.ru/api/video/person/796718/
https://rutube.ru/api/video/person/796718?format=xml
https://rutube.ru/api/video/person/796718?format=json
https://rutube.ru/api/video/person/796718/?format=xml
https://rutube.ru/api/video/person/796718/?format=json
please see what can be done

**Comment by spacecowboy** [comment]
Why would you expect that it would be able top arse that? Those results are not feeds.

**Reply by vejobo9414larland** [reply]
it's a pity(
I thought it was possible to add format recognition to receive profile updates in RSS format, similar to YouTube (https://www.youtube.com/feeds/videos.xml?user=...) and view data in the application.
thanks for the answer

**Reply by vejobo9414larland** [reply]
By the way, I consider the application one of the best (to put it bluntly, the best and most convenient) among free and open source RSS readers.
that I noticed. When updating all RSS channels, for some reason the process takes a relatively long time, the load on the network is not very large. This may be due to the specific firmware used on the phone.
Thank you very much for the application and work)
